### PR TITLE
Add hint for Compose React Components

### PR DESCRIPTION
**Pull request by angelobarbano**
Added hint and code example for "Compose React Components"

---

## ✅️ By submitting this PR, I have verified the following

- [x ] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x ] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x ] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/certifications/front-end-libraries/react/compose-react-components/index.md [hunk 1]
```diff
@@ -1,10 +1,32 @@
 ---
-title: Compose React Components
+title:Compose React Components
```
Comment (Asjas) [inline src/pages/certifications/front-end-libraries/react/compose-react-components/index.md]: Hi @angelobarbano 

You need to add a space between `:` and the first letter of the title. It's not rendering correctly. Please see [this page](https://github.com/freeCodeCamp/guide/blob/6883a27a4e3c2fe04ed9c39b4b71ed5b69c3b5c8/src/pages/certifications/front-end-libraries/react/compose-react-components/index.md) to see how it's rendering.

File: src/pages/certifications/front-end-libraries/react/compose-react-components/index.md [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
 ---
-## Compose React Components
+##Compose React Components
```
Comment (Asjas) [inline src/pages/certifications/front-end-libraries/react/compose-react-components/index.md]: Hi @angelobarbano 

You need to add a space between `##` and the first letter of the heading. It's not rendering correctly as well.

File: src/pages/certifications/front-end-libraries/react/compose-react-components/index.md (hunk 1 above)
Comment (Asjas) [inline src/pages/certifications/front-end-libraries/react/compose-react-components/index.md]: Hi @angelobarbano

You need to add a space between : and the first letter of the title. It's not rendering correctly. Please see [this page](https://github.com/freeCodeCamp/guide/blob/6883a27a4e3c2fe04ed9c39b4b71ed5b69c3b5c8/src/pages/certifications/front-end-libraries/react/compose-react-components/index.md) to see how it's rendering.

Comment (angelobarbano) [reply src/pages/certifications/front-end-libraries/react/compose-react-components/index.md]: Thanks again, fixed this as well.

**Comment by Asjas** [comment]
Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :tada:

**Pull request merged**